docs/RELEASING_KREW.md

```diff
@@ -14,11 +14,11 @@
     krew=out/bin/krew-darwin_amd64 # assuming macOS amd64
 
     KREW_ROOT="$(mktemp -d --tmpdir krew-XXXXXXXXXX)" KREW_OS=darwin \
-        $krew install --manifest=out/krew.yaml --archive=out/krew.tar.gz && \
+        $krew install --manifest=out/krew.yaml --archive=out/krew-darwin_amd64.tar.gz && \
     KREW_ROOT="$(mktemp -d --tmpdir krew-XXXXXXXXXX)" KREW_OS=linux \
-        $krew install --manifest=out/krew.yaml --archive=out/krew.tar.gz && \
+        $krew install --manifest=out/krew.yaml --archive=out/krew-linux_amd64.tar.gz && \
     KREW_ROOT="$(mktemp -d --tmpdir krew-XXXXXXXXXX)" KREW_OS=windows \
-        $krew install --manifest=out/krew.yaml --archive=out/krew.tar.gz
+        $krew install --manifest=out/krew.yaml --archive=out/krew-windows_amd64.tar.gz
```

Comment:
Can you please do something like this instead:

```sh
for osarch in darwin_amd64 darwin_arm64 linux_amd64 linux_arm linux_arm64 windows_amd64; do
   KREW_ROOT="$(mktemp -d --tmpdir krew-XXXXXXXXXX)" KREW_OS="${osarch%_*}" \
     $krew install --manifest=out/krew.yaml --archive="out/krew-${osarch}.tar.gz"
done
```

Reply:
I thought about it but didn't know if not everything was tested on purpose.

Reply:
Oh! It was on purpose to not mix architectures. Will add `KREW_ARCH`.